src/lib/extended/class/class.ts

```diff
@@ -24,16 +34,18 @@ export class ClassDirective extends BaseDirective2 implements DoCheck {
     this.setValue(val, '');
   }
 
-  constructor(protected elementRef: ElementRef,
-              protected styler: StyleUtils,
-              protected marshal: MediaMarshaller,
-              protected delegate: ɵNgClassImpl,
+  constructor(elementRef: ElementRef,
+              styler: StyleUtils,
+              marshal: MediaMarshaller,
+              iterableDiffers: IterableDiffers,
+              keyValueDiffers: KeyValueDiffers,
+              renderer2: Renderer2,
               @Optional() @Self() protected readonly ngClassInstance: NgClass) {
     super(elementRef, null!, styler, marshal);
     if (!this.ngClassInstance) {
       // Create an instance NgClass Directive instance only if `ngClass=""` has NOT been defined on
       // the same host element; since the responsive variations may be defined...
```

Comment:
It's not clear that this comment is still relevant? I don't see the check for the empty string not being defined?

Also there appears to be an extra `instance` in the first line of the comment.